shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/restaurant/RestaurantInfo.kt

```diff
@@ -0,0 +1,7 @@
+package org.itmodreamteam.myrest.shared.restaurant
+
+data class RestaurantInfo (
```

Comment:
I would use class named RestaurantInfo to return info about a restaurant. RestaurantRegistration and RestaurantUpdate would be the classes to provide data required for registration and update.

Reply:
What about providing phone, email, and website (optionally) right at the registration?